go.mod

```diff
@@ -54,3 +54,5 @@ require (
 	google.golang.org/grpc v1.24.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
+
+replace github.com/textileio/go-textile-core v0.0.1 => ../go-textile-core
```

Comment:
Will edit when [go-textile-core #30](https://github.com/textileio/go-textile-core/pull/30) is merged